src/server/routers.rs

```diff
@@ -52,16 +53,28 @@ async fn route(
         .route("/admin/accounts", get(self::admin::accounts::list))
         .route("/admin/accounts/:account", get(self::admin::accounts::get))
         .route("/admin/shares", post(self::admin::shares::post))
-        .route("/admin/tables", post(self::admin::tables::post))
-        .route("/admin/tables", get(self::admin::tables::list))
-        .route("/admin/tables/:table", get(self::admin::tables::get))
+        .route(
+            "/admin/shares/:share/schemas",
+            post(admin::shares::schemas::post),
+        )
         .route(
             "/admin/shares/:share/schemas/:schema/tables",
             post(admin::shares::schemas::tables::post),
         )
         .route_layer(middleware::from_fn(jwt::as_admin))
         .route("/admin/login", post(self::admin::login))
-        .layer(Extension(state.clone()));
+        .layer(Extension(state.clone()))
+        .layer(
+            CorsLayer::new()
+                .allow_origin(
+                    "http://localhost:3000"
```

Comment:
This is actually an oversight on my end, and this was meant to be temporary :). I'll remove that in #15. Generally speaking though, we may want to look into how we could handle CORS.

I have been working with a nextjs app where I invoke the APIs, while calling from a web app may always require CORS headers etc, in that I case I think I'll you proxy the calls via next's build in API mechanics.

For "raw" delta-sharing I see few use-cases, In this case though a management app would be nice. We may be able to use parts for that app I am building for this as well. In that case I would probably try and serve the bundle from the same server - so also no CORS...

If we decide to add it though, it should definately be configurable, otherwise its not quite useful :D.